packages/service/src/service/base.ts

```diff
@@ -35,7 +35,8 @@ export class BtcAssetsApiBase implements BaseApis {
     const packedParams = params ? '?' + new URLSearchParams(pickBy(params, (val) => val !== undefined)).toString() : '';
```

Comment:
`val => !! val` is more simple and clear?

Reply:
I think `val !== undefined` is better, if val is 0, it will be filtered out here

Reply:
Yes. The type of val is unknown